app/helpers/revisions_helper.rb

```diff
@@ -46,4 +46,67 @@ def link_to_conflicting_revision(record, revision)
     link_to text, record_revision_path(record, revision)
   end
 
+  def render_revision_object_icon(revision_object,object_type=nil)
```

Comment:
Not sure if this may work better in a presenter?

https://github.com/dradis/dradis-ce/tree/master/app/presenters